src/ex2/ItemManager.js

```diff
@@ -0,0 +1,44 @@
+import { PokemonClient } from './PokemonClient.js';
+
+export class ItemManager {
+  constructor() {
+    this.todos = [];
+  }
+
+  addItem(item) {
+    const id = Date.now();
+    const todo = {
+      id,
+      task: item,
+    };
+    this.todos.push(todo);
+    return todo;
+  }
+
+  removeItem(id) {
+    const filteredTodo = this.todos.filter((todo) => todo.id !== parseInt(id));
+    this.todos = [...filteredTodo];
+  }
+
+  removeAll(){
+    while (this.todos.length) {
+      this.todos.pop();
+    }
+  }
+
+  async addPokemons(ids) {
+    const pokemonsArray = ids.split(',');
```

Comment:
ids should be an array and the conversion of user input (the split..) should be outside.
Item manager should not be aware of the user input format and should not handle it's parsing as well

Reply:
fixed!